### ISSUE TITLE
This Version is No Longer Maintained, Use Fork 

### ISSUE THREAD

**Issue by innesst** [chrisknepper/android-messages-desktop#287]
I recommend you use this fork as it is actively maintained:

github.com/OrangeDrangon/android-messages-desktop

**Comment by jl2245** [comment]
THis is not verified, so everyone oyu take a risk of this being malware or recording all your text messages.

**Comment by innesst** [comment]
Unclear what makes you think that this Chris Knepper version is malware free? And why an active fork would be more likely to have malicious code?

**Comment by jl2245** [comment]
> Unclear what makes you think that this Chris Knepper version is malware free? And why an active fork would be more likely to have malicious code?

One installs in OSX with a developer being verified, the other does not, apps have to be submitted to apple for verification of code, others do not, that is why there is a difference.

**Comment by jl2245** [comment]
<img width="252" alt="Screen Shot 2022-04-01 at 12 55 06 PM" src="https://user-images.githubusercontent.com/15024740/161317018-41fa5944-f6a3-4198-b810-c9413f623397.png">
 This is what you see when you install Orange Dragons vs. Chris'

**Comment by LanikSJ** [comment]
> This is what you see when you install Orange Dragons vs. Chris'

All it means is the package isn't signed by the developer.  Apple charges 💰💰💰 to be able to sign your packages.  I don't imagine everyone is able to pay just to be able to release an app for Mac.

I scanned it several times and best I can tell it's safe to use.  The code is open source if you're not sure see for yourself.

**Comment by anonymousaga** [comment]
yes, I switched to this after the no-texts bug. It has an added plus - you can use it and one other device on android messages web simultaneously!

**Comment by fernandomora** [comment]
> THis is not verified, so everyone oyu take a risk of this being malware or recording all your text messages.

You can clone https://github.com/OrangeDrangon/android-messages-desktop repository and build yourself from the source code if you don't trust released package

A `yarn dist` should be enough to build it

**Comment by dmgolembiowski** [comment]
> > Unclear what makes you think that this Chris Knepper version is malware free? And why an active fork would be more likely to have malicious code?
> 
> One installs in OSX with a developer being verified, the other does not, apps have to be submitted to apple for verification of code, others do not, that is why there is a difference.

Thank you for pointing this out. Your observations are valuable but please be considerate of the human being on the other side of the screen. Even hackers who make poor choices deserve your common decency. :)
